### PR TITLE
feature: create sign in screen

### PR DESCRIPTION
**Pull request by Stukz**
This PR creates the sign in screen with

1. Header + Title
2. Text Inputs Fields widgets
3. Sign In Button

**FIGMA**
image

**CURRENT**
image

### CODE REVIEW

File: lib/presentation/ui/pages/login/login_view.dart [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  static const double _headerHeightPercentage = 0.3;
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      color: Colors.white,
+      child: Column(
+        children: [
+          Container(
+            width: MediaQuery.of(context).size.width,
```
Comment (huanachin) [inline lib/presentation/ui/pages/login/login_view.dart]: You can use double.maxFinite

**Review by huanachin — APPROVED** [review]
Nice work!

**Review by juanRodriguez17 — APPROVED** [review]

File: lib/presentation/ui/pages/login/login_view.dart [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                MediaQuery.of(context).size.height * _headerHeightPercentage,
+            decoration: const BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage('assets/images/header.png'),
+                fit: BoxFit.fill,
+              ),
+            ),
+            child: const Center(
+              child: Text(
+                'TARGET MVD',
```
Comment (amaury901130) [inline lib/presentation/ui/pages/login/login_view.dart:29]: don't use plain strings, move this to the intl file and use S.of(context) to get the value.
same for others.

File: lib/presentation/ui/pages/login/login_view.dart (hunk 1 above)
Comment (Stukz) [reply lib/presentation/ui/pages/login/login_view.dart]: yes, thanks!

**Pull request merged**